scoop/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG version=ltsc2016
+FROM mcr.microsoft.com/windows/servercore:$version
+
+ENV SCOOP "C:\scoop"
+ENV SCOOP_HOME "C:\scoop\apps\scoop\current"
+
+RUN powershell.exe -NoLogo -Command " \
+        iwr -UseBasicParsing 'https://get.scoop.sh' | iex; \
+        scoop install 7zip git sudo innounp lessmsi dark; \
```

Comment:
I would like to only provide a base image, otherwise people demand more and more tools pre-installed :-)